app/data/formPages/types.ts

```diff
@@ -1,14 +1,20 @@
+import { Dispatch, SetStateAction } from "react";
+
+export interface SummaryFormProps<TProjectRevision> {
+  projectRevision: TProjectRevision;
+  viewOnly?: boolean;
+  isOnAmendmentsAndOtherRevisionsPage?: boolean;
+  setHasDiff?: Dispatch<SetStateAction<boolean>>;
```

Comment:
There is no need to carry these react artifact types around in our code. This can be a `(hasDiff: boolean) => void;` type